tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -266,6 +267,7 @@ public function testSupportsAllConfigFormats(): void
                         'cache_key_generator' => null,
                         'respect_response_cache_directives' => ['X-Foo'],
                         'blacklisted_paths' => ['@/path/not-to-be/cached@'],
+                        'cache_listeners' => [],
```

Comment:
can we do the full example with 2 cache listeners configured?

Reply:
y0... I am kind of eager, if this is, what you expected. wörks for me so far.